.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: Snyk
+
+on:
+  push:
+    branches: [ main, jd-snyk-action ]
+  workflow_dispatch:
+
+jobs:
+  snyk:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+
+      - name: SBT - Run Snyk to check for vulnerabilities
+        uses: snyk/actions/scala@0.3.0
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          args: --org=the-guardian-cuu --project-name=${{ github.repository }} --file=./build.sbt
```

Comment:
I think the `file` argument is redundant as that's the default value for SBT

Reply:
if you miss it out, does it not go off and find some other stuff?  I did have to add that to the promos one but maybe I was mistaken

Reply:
"When omitted Snyk will try to detect the appropriate file for your project."

I think it can work if you only have a build.sbt but no other package.json etc, but I prefer to be explicit, I think in this case it won't work right.

https://docs.snyk.io/features/snyk-cli/guides-for-our-cli/cli-reference#:~:text=When%20omitted%20Snyk%20will%20try%20to%20detect%20the%20appropriate%20file%20for%20your%20project.

Reply:
Perhaps it searches for build files if you don't tell it.  I'm not sure so ignore this comment.